app/assets/javascripts/components/student/pages/lessons_page.es6.jsx

```diff
@@ -1,12 +1,35 @@
 class LessonsPage extends React.Component {
```

Comment:
Currently the admin lessons page is also called LessonsPage.. Can we change this to StudentLessonsPage